test-utils/substrate-test-runner/Cargo.toml

```
@@ -0,0 +1,56 @@
+[package]
+name = "substrate-test-runner"
+version = "0.8.0"
+authors = ["Tomasz Drwięga <tomusdrw@gmail.com>", "Seun Lanlege <seunlanlege@gmail.com>"]
```

First is always the admin-url. You can add others, if you want your email-addr exposed, but we don't usually do that

//cc @tomusdrw 
```suggestion
authors = ["Parity Technologies <admin@parity.io>", "Tomasz Drwięga <tomusdrw@gmail.com>", "Seun Lanlege <seunlanlege@gmail.com>"]
```